### ISSUE TITLE
JUL bridge generates a lot of unnecessary data

### ISSUE THREAD

**Issue by andrzejressel** [zio/zio-logging#876]
Currently due to ".level = ALL"  in Jul config all logs go to zio. I was expecting some kind of overhead, but not too much. When profiling my JAXB app, 20% of deserialization time and 90% of memory usage is wasted due to pushing FINEST level logs.

I guess current api has to be deprecated and every method should also have parameter with JUL config.

**Comment by justcoon** [comment]
hi @andrzejressel 
this seems to be similar problem like https://github.com/zio/zio-logging/issues/809, which was solved with https://github.com/zio/zio-logging/pull/810, and also [jul-bridge](https://github.com/zio/zio-logging/blob/master/jul-bridge/src/main/scala/zio/logging/jul/bridge/ZioLoggerRuntime.scala#L26) have similar support

so if this is something like that, than it should be possible to set `LogFilter` for `jul-bridge` as mentioned in [docs]( https://github.com/zio/zio-logging/blob/master/docs/jul-bridge.md)

**Comment by andrzejressel** [comment]
Yeah, that seems to fix the issue. I think I will deprecate initialize and add note about finest level.

**Comment by 987Nabil** [comment]
@andrzejressel is this issue still needed?
